src/test/java/cucumber/steps/Hooks.java

```diff
@@ -0,0 +1,33 @@
+package cucumber.steps;
+
+import api.ApiClient;
+import exeptions.FailedToLoginException;
+import helpers.UserTokenGenerator;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Hooks {
+    private static final Logger logger = LogManager.getLogger();
```

Comment:
Maybe the name of the constant should be in upper case?

Reply:
done